src/redis-cli.c

```diff
@@ -1387,6 +1389,12 @@ static int parseOptions(int argc, char **argv) {
         fprintf(stderr,"Try %s --help for more information.\n", argv[0]);
         exit(1);
     }
+
+    if (!config.no_auth_warning) {
```

Comment:
Hi, shouldn't this also test that config.auth is not NULL?

Reply:
Yes, i forgot it. Fix soon :-)